sparkbox/designsystemssurvey
# Setup 2019 site

### Description
* Move 2018 project into a 2018 directory
* Add gatsby starter template
* Update info in package.json to replace starter info
* Add node sass
* Update 2018 readme, add readme for repo, 2019 site
* Update metadata in gatsby config

### To test
Use the readme in the 2019 directory to get the project running. Check that you see some Gatsby.js starter pages on localhost:8000
